src/ocean/net/server/unix/UnixConnectionHandler.d

```diff
@@ -100,7 +124,12 @@ public class BasicCommandHandler
         void delegate ( cstring ) send_response,
         void delegate (ref mstring) wait_reply)
     {
+
         if (auto handler = command in this.handlers)
+        {
+            (*handler)(args, send_response);
+        }
+        else if (auto handler = command in this.interactive_handlers)
```

Comment:
As the interactive session is a more "advanced" API, shouldn't it preferred ?
Also the order in which they are processed should be documented somewhere.

Reply:
I think what I really want to do is to iterate over keys of both maps in the constructor and verify that there are no maches. `O(mn)` complexity should not matter in case like this (I hope).

Reply:
That was something else I was wondering if we could do, but I'm usually on the opinion that we should be as lax as practical. So we could have a natural overriding scheme. But I don't have a strong opinion here.

Reply:
Just out of the interest, why do you think that the more advanced API should have the priority in the overriding? I would thought it's the reverse.

Reply:
Since the more advanced API is a superset of the simpler API, I expect users of the more advanced API to support a superset of the features supported by the simpler API, because it's usually trivial to wrap the simpler API.

Reply:
Yeah, I think what you're saying makes sense. I'll update the ordering and the documentation.